examples/config.yaml

```diff
@@ -0,0 +1,65 @@
+machines:
+
+  - name: ct-23-0001 #1st machine
+    dataset:
+      tags: #list of tags for 1st machine
+        - GRA-TE  -23-0733.PV
```

Comment:
Unfortunately, we cant put tag names out there (yet). I know we may have some already, but they are not open.